internal/db/schema/migrations/postgres/14/01_event.up.sql

```diff
@@ -0,0 +1,217 @@
+'the domain';
+
+create table event_sink_type_enm (
+    name text primary key
+        constraint only_predefined_event_sink_types_allowed
+        check (
+            name in (
+                'stderr',
+                'file'
+            )
+        )
+);
+comment on table event_type_enm is
+'event_sink_type_enm is an enumeration table for the valid sink types within '
+'the domain';
+
+create table event_format_type_enm (
+    name text primary key
+        constraint only_predefined_event_format_types_allowed
+        check (
+            name in (
+                'json',
+                'text',
+                'cloudevent',
+                'hclogtext',
+                'hclogjson'
+            )
+        )
+);
+comment on table event_format_type_enm is
+'event_format_type_enm is an enumeration table for the valid event format types '
+'within the domain';
+
+create table event_config (
+    public_id wt_public_id primary key,
+    scope_id wt_scope_id not null
+        constraint iam_scope_global_fkey
+            references iam_scope_global(scope_id)
+            on delete cascade
+            on update cascade,
+    constraint scope_id_uq -- only allow one config per scope
+        unique (scope_id),   
+    name wt_name,
+    description wt_description,
+    create_time wt_timestamp,
+    update_time wt_timestamp,
+    version wt_version
+);
+comment on table event_config is
+'event_config is a table where each entry defines the event configuration for '
+'a scope.  Currently, the only supported scope is global';
+
+create table event_type_enabled (
+    config_id wt_public_id
+        constraint event_config_fkey
+            references event_config(public_id)
+            on delete cascade
+            on update cascade,
+    event_type text not null
+        constraint event_type_enm_fkey
+            references event_type_enm (name)
+            on delete restrict
+            on update cascade,
+    --constraint config_id_event_type_uq
+        --unique (config_id, event_type) -- only allow an event type to be enable once
+        --do we still need this ^ if we have what is below?
+    primary key (config_id, event_type)
+);
+comment on table event_type_enabled is
+'event_type_enable is a table where each entry represents that eventing has '
+'been enabled for the specified event type in an event configuration';
+
+
+create table event_sink(
+    public_id wt_public_id primary key,
+    config_id wt_public_id 
+        constraint event_config_fkey
+        references event_config(public_id)
+        on delete cascade
+        on update cascade,
+    event_type text not null
+        constraint event_type_enm_fkey
+        references event_type_enm(name)
+        on delete restrict
+        on update cascade,
+    sink_type text not null
+        constraint event_sink_type_enm_fkey
+        references event_sink_type_enm(name)
+        on delete restrict
+        on update cascade,
+    format_type text not null
```

Comment:
Can an event config have multiple sinks for the same event type, format type, and event type? If so, why?